:sparkles: feat: Add 'Improve Content' section to summary (#159)
OurTechCommunity/catchup: summary/combined-summary.adoc

// include autogenerated combined summaries
include::autogenerated-combined-summary.adoc[]

> [.improve-heading]

Reviewer: ```suggestion
[#improve-content]
```

If we use an id instead of a class here, then we can directly reference the section with `{url}#improve-content`

Want to improve the content of this page?

[.improve-buttons]
* link:https://github.com/OurTechCommunity/catchup/tree/main/summary/sessions[Edit these summaries on GitHub.^]

> * link:https://github.com/OurTechCommunity/catchup/issues/new?title=Improve%20combined%20summary&body=Enter%20changes%20to%20be%20made[Report the content issue.^]

Reviewer 2: Just adding a bit more details to the body and some styling.

```suggestion
* link:https://github.com/OurTechCommunity/catchup/issues/new?title=Improve%20Combined%20Summary&body=%3E%20Please%20describe%20the%20issue%20with%20the%20summary%20content%20and%20the%20changes%20to%20be%20made.[Report the content issue.^]
```

> * link:https://github.com/OurTechCommunity/catchup/issues/new?title=Improve%20Combined%20Summary&body=%3E%20Please%20describe%20the%20issue%20with%20the%20summary%20content%20and%20the%20changes%20to%20be%20made.[Report the content issue.^]

Reviewer: ```suggestion
```

Do we need to worry about content issues?
Content issues (outdated, deprecated, incorrect information) can appear on docs pages, but I don't think they apply here.

Reviewer 3: Oh alright. I mean Content in this context could literally mean, the content of the summary.

We can rename that Issue. So, they can report any issue; if they do not want to edit the summaries directly and just point out any issue.

What do you think? I think it is a valid `Action`.

Reviewer: Hmm ok, that would make more sense.

> * link:https://github.com/OurTechCommunity/catchup/issues/new?title=Improve%20Combined%20Summary&body=%3E%20Please%20describe%20the%20issue%20with%20the%20summary%20content%20and%20the%20changes%20to%20be%20made.[Report the content issue.^]

Reviewer: Slight issue with having this at the bottom of the combined summary page, the footer will almost never be seen.
We need to move this either to the sidebar or at the beginning.

Reviewer: ![image](https://github.com/OurTechCommunity/catchup/assets/25801283/4d788b08-f29c-41e3-8887-3aea20c07dff)

*scrolling*

https://github.com/OurTechCommunity/catchup/assets/25801283/5a010978-eff0-48bf-8c65-091a308cba1f

*more scrolling*

![image](https://github.com/OurTechCommunity/catchup/assets/25801283/ea6449da-4bd2-4eab-a5b3-a5f4052ecb52)

Reviewer: @sushma1031 can you please change the table of contents to be in the left sidebar, and add a heading that jumps to the bottom of the page?
The TOC position is a single configuration setting: https://docs.asciidoctor.org/asciidoc/latest/toc/position/
Creating a new heading can be done with `== Heading`

Author: All right, I'll work on it

> * link:https://github.com/OurTechCommunity/catchup/blob/main/summary/combined-summary-template.adoc[View the source on GitHub.^]

Reviewer 2: We should link the content of the sessions as we did with the individual sessions, not the template.

```suggestion
* link:https://github.com/OurTechCommunity/catchup/tree/main/summary/sessions[View the source on GitHub.^]
```
